web/src/lib/assets/sun.svelte

```diff
@@ -0,0 +1,14 @@
+<script lang="ts">
```

Comment:
Are there not existing md* svg paths to use here instead?

Reply:
Sorry what are `md* svg paths` ?

Reply:
The way we use icons everywhere else in the app

Reply:
I tried to import the svg in an `img` tag like it's done, but you're loosing the `fill="currentColor"` property

Reply:
I'm pretty sure you can set that with our icon component.

Reply:
Oh sorry I just realized you were talking about that component.